frontend/src/app/home/home.page.ts

```diff
@@ -1,8 +1,25 @@
 import { Component } from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+import { NavController } from '@ionic/angular';
 
 @Component({
   selector: 'app-home',
   templateUrl: 'home.page.html',
   styleUrls: ['home.page.scss'],
 })
-export class HomePage {}
+export class HomePage {
+  checkOutForm = this.formBuilder.group({ firstName: [''], lastName: [''] });
+
+  constructor(
+    private formBuilder: FormBuilder,
+    private navController: NavController
+  ) {}
+  myS(): void {
```

Comment:
For future reference: You might want to assign better names for your methods, and maybe add some comments telling what they do. This is no problem at the moment, since this is really just a "hello world" application.

Reply:
Yeah, thought this will be deleted anyway, so i didn't had an eye on it, but for the future i will get better names  :)